index.html

```diff
@@ -31,22 +46,48 @@ <h1 class="author">I’m Tanveer Glad to see you!</h1>
             <li><a href="#"><img src="./assets/Vector-2.svg" alt="m-icon"></a></li>
             <li><a href="#"><img src="./assets/Vector-1.svg" alt="github"></a></li>
             <li><a href="#"><img src="./assets/Vector-3.svg" alt="victory"></a></li>
-          </ul>
+          </ul>
         </div>
+      </div>
```

Comment:
The design is fantastic, and you can improve it by following the same Figma design rules. So kindly make sure the bottom-left corner of the header is rounded.

![image](https://user-images.githubusercontent.com/34779928/200075159-22034e6b-6dd0-4c76-abc8-23bcd6f9eccd.png)